lib/ansible/module_utils/redfish_utils.py

```diff
@@ -683,17 +703,26 @@ def set_one_time_boot_device(self, bootdevice):
         else:
             payload = {"Boot": {"BootSourceOverrideTarget": bootdevice}}
 
-        response = self.patch_request(self.root_uri + self.systems_uri, payload, HEADERS)
+        response = self.patch_request(self.root_uri + systems_uri, payload, HEADERS)
         if response['ret'] is False:
             return response
         return {'ret': True}
 
-    def set_bios_attributes(self, attr):
+    def set_multi_time_boot_device(self, bootdevice):
+        ret = True
+        entries = []
+        for systems_uri in self.systems_uris:
+            boot_device = self.set_one_time_boot_device(bootdevice, systems_uri)
+            ret = boot_device.pop('ret') and ret
+            entries.append(boot_device['entries'])
+        return dict(ret=ret, entries=entries)
+
+    def set_bios_attributes(self, attr, systems_uri):
```

Comment:
This is a "set" method called from redfish_config (which doesn't pass in the systems_uri param). So need to remove that param here and just use self.system_uris[0] below.

Reply:
Done